src/js/view/progressbar.js

```diff
@@ -28,6 +28,14 @@ export default function ProgressBar(elements) {
                 durationElement.innerText = '--:--';
                 sliderElem.style.width = '0%';
                 mediaDuration = 0;
+            } else if (utils.streamType(duration) === 'LIVE') {
+                let durationChanged = mediaDuration != duration;
+                if (durationChanged) {
+                    mediaDuration = duration;
+                    elapsedElem.innerText = 'Live';
+                    durationElement.innerText = '';
+                    sliderElem.style.width = '100%';
```

Comment:
@esboyd @egreaves For the control bar during live streams, I set the text simply to 'Live' and show a full progress bar (instead of leaving a lot of whitespace).
[
![img_20170327_141459](https://cloud.githubusercontent.com/assets/584053/24371180/0eb2b1ee-12f8-11e7-808b-ba9c9109ff0b.jpg)
](url)